runner.md

```diff
@@ -4,7 +4,7 @@ title: Runner role
 ---
 # Runner role
 
-The runner role switches off between team members each week.  The runner should be somebody who is planning to be in for the whole week.
+The runner role switches off between team members each week.  The runner should be somebody who is planning to be in for the whole week. The runner as executes a deployment of all our applications to staging and production during the course of their week as the runner. 
```

Comment:
Didn't we say it was the Board Manager who would deploy?  Since the runner already has a lot to do?

Reply:
That's what I remember as well.

Reply:
I moved the docs over to the board handbook and added a section about the Board Manager/Deployments. Let me know how this looks.